src/pages/terms-of-use/index.tsx

```diff
@@ -215,6 +216,78 @@ const TermsOfUse = () => {
                     <ExternalLink url="https://www.lse.ac.uk/granthaminstitute/cclw-terms-and-conditions">View</ExternalLink>
                   </td>
                 </tr>
+                <tr>
+                  <td>
+                    <ExternalLink url="https://unfccc.int/topics/global-stocktake/information-portal?field_document_topic_target_id=All&field_document_type_target_id=4174&field_calculated_author_type_target_id=All&field_document_ca_target_id=">
+                      Global Stocktake Information Portal
+                    </ExternalLink>
+                  </td>
+                  <td>Submissions to the first Global Stocktake by parties and non-Party stakeholders </td>
+                  <td>23 May 2023</td>
+                  <td>
+                    <ExternalLink url="https://unfccc.int/this-site/terms-of-use">View</ExternalLink>
```

Comment:
This link refers to "This website" - I don't think this statement protects their information on our website @henry-cpr 

Ditto for the rest of the links

Reply:
The relevant paragraph of note for our usage of documents from the link is 
"All official texts, data and documents, including low resolution webcast transmissions, are in the public domain and may be freely downloaded, copied and printed provided no change to the content is introduced, and the source is duly acknowledged."

Reply:
Closed https://linear.app/climate-policy-radar/issue/SOF-55/review-the-use-of-terms-and-conditions-of-third-party-sites